CHANGES.rst

```diff
@@ -8,48 +8,178 @@ https://hyperspy.readthedocs.io/en/latest/user_guide/changes.html
 
 .. towncrier-draft-entries:: |release| [UNRELEASED]
 
+.. towncrier release notes start
+
+.. _changes_2.0:
+
+2.0rc0 (2023-11-10)
+===================
+
+Release Highlights
+------------------
+- Hyperspy has split off some of the reading/writing files and domain specific functionalities into separate libraries!
+  
+  - `RosettaSciIO <https://hyperspy.org/rosettasciio>`_: A library for reading and writing scientific data files.
+    See `RosettaSciIO release notes <https://hyperspy.org/rosettasciio/changes.html>`_ for new features.
+  - `exSpy <https://exspy.readthedocs.io>`_: A library for EELS and EDS analysis.
+    See `exSpy release notes <https://hyperspy.org/exspy/changes.html>`_ for new features.
+  - `HoloSpy <https://holospy.readthedocs.io>`_: A library for analysis of (off-axis) electron holography data.
+    See `HoloSpy release notes <https://holospy.readthedocs.io/en/latest/changes.html>`_ for new features.
+
+- The :py:mod:`~.api.plot.markers` API has been refactored
+  - Lazy markers are now supported
+  - Plotting many markers is now _much_ faster
+  - Support for :py:class:`~.api.plot.markers.Polygons` has been added
+- The documentation has been restructured and improved!
+  - Short example scripts are now included in the documentation
+  - Improved guides for lazy computing as well as an improved developer guide
+- Plotting is easier and more consistent
+  - Horizontal plotting is now supported in the ``ipympl`` and ``widget`` backends
+  - Changing navigation coordinates using keyboard ``Arrow`` have been removed, use
+  ``Crtl`` + ``Arrow`` instead.
+  - Jump to navigation position using ``shift`` + click in the navigator.
+- HyperSpy now works with Pyodide/Jupyterlite, checkout `hyperspy.org/jupyterlite-hyperspy <https://hyperspy.org/jupyterlite-hyperspy>`_!
+
```

Comment:
Mention API changes/deprecation as another bullet in the highlights, just to make people aware?